### DISCUSSION TITLE
using the api to remove an application

### DISCUSSION THREAD

**Discussion by darren-rose** [argoproj/argo-cd#10282] [Q&A]
Hi, I am attempting to use the api to remove an application, which works, but when I try and update the associated project removing the source repo and destination of the removed application I receive a 400 response. When I also try and remove the associated repository I receive 404 responses. Any help using the api to remove an application its repository and removing it's entries in the associated project would be appreciated.

**Comment by pasha-codefresh** [comment]
Could you please add details? what calls are you doing and what is response
